_pages/clients/cli.md

```diff
@@ -106,11 +106,15 @@ Run `algo run --help` to see more command options or view the following [Options
 #### Input Data Options
 There are several options for specifying the type and source of input data. The Algorithmia CLI supports JSON, text, and binary data, as well as an option to auto-detect the data type.
 
+<div class="syn-table-container" markdown="1">
+
```

Comment:
![Screen Shot 2019-11-27 at 8 29 20 AM](https://user-images.githubusercontent.com/8980868/69741528-104e8c00-10f0-11ea-8070-1a24bba6cdcf.png)
The left column is a bit cramped. If we add a width of about 30% to the first `<th>` element it's easier to read. Since this is generated via markdown, we could target this with a new class on the table container:

```
<div class="syn-table-container syn-table-col" markdown="1">

.syn-table-col th:first-child {
  // magic
}
```

Reply:
Updated!